be/src/common/config.h

```diff
@@ -383,6 +383,17 @@ CONF_Int32(be_exit_after_disk_write_hang_second, "60");
 // turn off dictionary dictionary encoding. This only will detect first chunk
 // set to 1 means always use dictionary encoding
 CONF_Double(dictionary_encoding_ratio, "0.7");
+
+// Some data types use dictionary encoding, and this configuration is used to control
+// the size of dictionary pages. If you want a higher compression ratio, please increase
+// this configuration item, but be aware that excessively large values may lead to
+// performance degradation.
+CONF_Int32(dictionary_page_size, "1048576");
```

Comment:
can this config item be runtime mutable? restart will be very heavy for just configvar adjustment.

Reply:
@kevincai Each type of encoder object is created during BE startup. When creating the encoder, the sizes of data page size/dict page size are already determined, it's not easy to configure them dynamically. Additionally, in my opinion, for practical use cases, page size is not a frequently changing configuration option. Therefore, I think it's ok to make this configuration unchangeable.